.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+docker-compose.yml
```

Comment:
Nah, don't put the token in there. Always pull the token from an environment variable, and add it to an `env.sh` script that sets up the environment. `env.sh` can do some fancy stuff like download the token from a secure place with aws credentials, or look for a special file that we don't commit to the repo.